docs/pages/Kratos/Expressions/Utilities/Comb.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Comb
+keywords: 
+tags: [combing, expressions]
+sidebar: kratos_expressions
+summary: 
+---
+
+## Introduction
+This allows combining multiple expressions in to one expression. **The number of entities on all the combining expressions should have the same.**
+
+Example:
+All provided expressions in the list_of_expression.
+```
+For example, let the list_of_expression contain the following expressions:
+exp_list[0] = data{1, 2, 3} with 3 items, and item shape = []
```

Comment:
just curious, if shape [ ] takes 1 item, shape = [3] takes 3 items. What should I aspect with shape = [1] and shape = [2]

Reply:
`shape[1]` is not allowed. it will throw an error in the `Expression` if it find `shape[1]`. `shape[2]` cannot be used within Kratos since we don't have 2D vectors. But `shape[2]` can be used with `Expressions`, but won't be able to convert to `Kratos::Variable`

Reply:
`shape[1]` and `shape[]` both represent scalar quantities.